src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandom.java

```diff
@@ -34,26 +34,28 @@ public void adsFandomAdsDesktop(String pageType, String pageName) {
   )
   public void adsFandomAdsMobile(String pageType, String pageName) {
     AdsFandomObject fandomPage = loadPage(pageName);
-    verifySlots(fandomPage, pageType);
+    verifySlots(fandomPage, pageType, AdsFandomContent.TOP_BOXAD_MOBILE);
   }
 
-  private void verifySlots(AdsFandomObject fandomPage, String pageType) {
+  private void verifySlots(AdsFandomObject fandomPage, String pageType, String slotName) {
     if (pageType.equals(AdsFandomTestTemplate.PAGE_TYPE_ARTICLE)) {
-      verifyArticleSlots(fandomPage);
+      verifyArticleSlots(fandomPage, slotName);
```

Comment:
let's rethink that together because it looks a little bit odd - you pass slotName but verify more than one slot.